tests/test_sig_stfl_mem.c

```diff
@@ -0,0 +1,225 @@
+// SPDX-License-Identifier: MIT
+
```

Comment:
I realize now this file (and the other test files) should have some comments at the top of the file explaining what this test file is for.